knative-operator/deploy/resources/knativekafka/controller/1-eventing-kafka-controller.yaml

```
@@ -950,6 +950,8 @@ spec:
       containers:
         - name: controller
           image: TO_BE_REPLACED
+          args:
+            - "--disable-controllers=broker-controller, trigger-controller, sink-controller"
```

Spaces aren't allowed:
```suggestion
            - "--disable-controllers=broker-controller,trigger-controller,sink-controller"
```